content/applications/productivity/whatsapp.rst

```diff
@@ -0,0 +1,569 @@
+================
+WhatsApp in Odoo
+================
+
+WhatsApp is a messaging service operated by Meta. It is commonly used as a communication tool in
+many countries and within many businesses. This documentation will cover the integration of a
+WhatsApp Business Account with Odoo.
```

Comment:
Can you reuse the intro on the current page (and improve it if you'd like)?
https://github.com/odoo/documentation/blob/saas-16.3/content/applications/productivity/whatsapp.rst?plain=1

IMO, it better distinguishes WhatsApp, Odoo WhatsApp and WhatsApp Business Account. The list also explains straight away what's doable with this app.

Reply:
I've moved the page over! Thanks for the suggestion :)